docs/basics/integrations/webhooks.md

```diff
@@ -71,3 +113,21 @@ Each event sends a POST request containing the job object with the following dat
 | `build`              | STRING    | A custom value that can group multiple related tests.                                                                                                                                                                                                                                |
 | `automation_backend` | STRING    | The framework on which the test is run. Currently this field is not supported for RDC events and it will always appear as an empty string.                                                                                                                                           |
 | `data_type`          | ENUM      | The type of event that triggered the webhook. Possible values are: <br/><ul><li>`vdc`</li><li>`rdc`</li></ul>                                                                                                                                                                        |
+
+
+## Visual Testing Build completed event
+
+| Data Field          | Format | Required | Description                                                                                                                                                                           |
+|---------------------|--------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `org_id`            | STRING | TRUE     | Id of build owner's Sauce Labs organization.                                                                                                                                          |
+| `user_id`           | STRING | TRUE     | Build owner's Sauce Labs user id.                                                                                                                                                     |
+| `team_ids`          | ARRAY  | TRUE     | A list of Sauce Labs team ids which the build owner belongs to.                                                                                                                       |
```

Comment:
Hmm, I see this is an array but code suggests it will only ever be a single entry in this array. Do you think it's worth having this as an array vs just the owning team ID?

Reply:
We had a chat and decided to use only and active team.